debian/rules

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/make -f
 
+export DEB_BUILD_MAINT_OPTIONS=hardening=+all
+
 %:
```

Comment:
I'm nervous about this - libibverbs is a performance library, adding security check overhead is probably not desired. Do you know how much overhead there is?